src/main/scala/nl/knaw/dans/easy/ingest/EasyIngest.scala

```diff
@@ -208,12 +215,19 @@ object EasyIngest {
       System.exit(13)
     }
 
+  private class CompositeException(throwables: List[Throwable]) extends RuntimeException(throwables.foldLeft("")((msg, t) => s"$msg\n${getMessage(t)} ${getStackTrace(t)}"))
```

Comment:
I know this class also originally was `private`, but isn't that bad practice here. You expose instances of `CompositeException` to the 'outside world', right? This way (if I'm correct - and I've not tested this!!!) you're not able to test for these exceptions or pattern match on them elsewhere.